docs/conf.py

```diff
@@ -72,7 +72,7 @@
 def _correct_signatures(app, what, name, obj, options, signature, return_annotation):
     if what == "class":
         signature = str(inspect.signature(obj.__init__)).replace("self, ", "")
-    return (signature, return_annotation)
+    return signature, return_annotation
```

Comment:
I'd say that casting as a tuple at line of return is more explicit than knowing that any comma-separated returns do so automatically

Reply:
I'm used to the simpler non-parens syntax, and this is suggested by PyCharm's auto checker. I don't feel that strongly about it though. We can change it back if you want.

Reply:
I would probably change it back. One of those style things that yes, saves characters in a line, but avoids any potential confusion especially in cases of returning nested tuples or tuple-likes such as arraays (which you'd then have to know are nested at a particular level; classic example of this is `np.where`)